.ddev/commands/cypress/percy-run

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#ddev-generated
+## Description: Use Cypress in "runner" mode.
```

Comment:
Is this also copy/paste? I am guessing this is doing more than just running Cypress.

Reply:
Sorry, my bad. Yes, it's a copy-paste. I have updated the doc comment now.